tests/net/bridge/testcase.yaml

```diff
@@ -0,0 +1,10 @@
+tests:
+  net.eth_bridge:
+    min_ram: 16
+    tags: net bridge
+    depends_on: netif
+  net.eth_bridge.ip:
+    min_ram: 16
+    tags: net bridge
+    depends_on: netif
+    extra_args: CONF_FILE="prj_ip.conf"
```

Comment:
Please change this to
```
common:
  min_ram: 16
  tags: net bridge
  depends_on: netif
tests:
  net.eth_bridge:
    extra_configs:
      - CONFIG_NET_IPV4=n
      - CONFIG_NET_IPV6=n
  net.eth_bridge.ip:
    extra_configs:
      - CONFIG_NET_IPV4=y
      - CONFIG_NET_IPV6=y
```
The extra config in `net.eth_bridge` is duplicate (as it is already in prj.conf file) but something is needed in that section as the section cannot be left empty. If done this way, there is no need to have extra prj config file and we need to maintain only one prj file in the tests.